### DISCUSSION TITLE
Bind mounting trmm-backend, yes or no?

### DISCUSSION THREAD

**Discussion by ninjamonkey198206** [amidaware/tacticalrmm#766] [Q&A] [answered]
Is there any reason why we shouldn't bind mount trmm-backend, or any of the volumes for that matter?

**Comment by sadnub** [comment, marked as answer]
What would the benefit be? Just access to the files and data on the docker host?

**Reply by ninjamonkey198206** [reply]
Yes, for something I wanted to try out, but I found another way.

**Reply by sadnub** [reply]
You can write your own docker-compose and use the tactical container images anyway you need. If you need more options exposed in the image, we can add that. The stock docker-compose is to just get a standard server up and running

**Reply by ninjamonkey198206** [reply]
@sadnub No worries. I was just trying to figure out a way to automate cert management when the cert renewal is handled by the router or host.

I think I've got it figured out, I just need to sit down, write the scripts, and test it.

As far as the docker-compose file goes, in the other discussion about HAProxy I listed some potential changes towards the end to prevent issues I ran into, such as the network gateways sometimes not being configured/missing (weird), and adding names to the networks via driver opts to make firewall rules and troubleshooting easier.

They don't hurt anything by being included, but not having them occasionally caused headaches for me.

**Reply by ninjamonkey198206** [reply]
Basically just QoL improvements to make things easier.
